### DISCUSSION TITLE
Exporting to pdf

### DISCUSSION THREAD

**Discussion by StefanBRas** [mkdocstrings/mkdocstrings#473] [Q&A]
Hi,

In addition to html, I need to generate a single pdf with the full API reference.
Im trying to use either the [pdf-export](https://github.com/zhaoterryy/mkdocs-pdf-export-plugin) or [with-pdf](https://github.com/orzih/mkdocs-with-pdf) but it's not really working.

I'm new to mkdocs and having issues with exactly how the plugins interact, so it's difficult for me to come up with a solution. It seems to me that no matter the order of the plugins, the generated html from mkdocstrings is never included in the export documents.

Im open to all kind of solutions, and having separate mkdoc.yml for html and pdf is also fine.

Im thinking to either:

- not use a pdf-export plugin, but rather just make a single html page with the whole API reference and then use external html-to-pdf tools
- make a custom handler that creates "physical files" with either markdown or html instead of inlining it, so it's caught by the pdf export plugins in a second run

Looking forward to hearing your thoughts! 

Edit: 
It looks like something like this could maybe work: https://github.com/mkdocstrings/mkdocstrings/discussions/346

**Comment by pawamoy** [comment]
I've created a MkDocs plugin that outputs a manual page of the documentation (manpage) using Pandoc. Maybe the same approach could be used to output a PDF as well. The project is only available to my sponsors yet: https://pawamoy.github.io/mkdocs-manpage/

**Reply by StefanBRas** [reply]
Thanks for letting me know. I did the thing from https://github.com/mkdocstrings/mkdocstrings/discussions/346#discussioncomment-1675340 and it worked well. I don't work at that place anymore though, so not my problem lol.
